lib/netutil.py

```diff
@@ -1,5 +1,20 @@
 import socket
 
+from lib.commands import local_cmd
+from lib.common import wait_for
+
+def wait_for_tcp_port(host, port, port_desc, ping=True, host_desc=None):
+    if host_desc:
+        host_desc = f"({host_desc})"
```

Comment:
what about getting the formatting space here rather than logging a trailing one when not set?